client/web/src/components/fuzzyFinder/FuzzyModal.tsx

```diff
@@ -143,24 +154,23 @@ export const FuzzyModal: React.FunctionComponent<FuzzyModalProps> = props => {
 function plural(what: string, count: number, isComplete: boolean): string {
     return count.toLocaleString() + (isComplete ? '' : '+') + ' ' + what + (count === 1 ? '' : 's')
```

Comment:
Consider using a template string instead of string concatenation.
```suggestion
    return `${count.toLocaleString()}${isComplete ? '' : '+'} ${pluralize(what, count)}`
```